plugins/modules/aci_route_control_context.py

```diff
@@ -0,0 +1,312 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "certified"}
+
+DOCUMENTATION = r"""
+---
+module: aci_route_control_context
+short_description: Manage Route Control Context objects (rtctrl:CtxP)
```

Comment:
I've re-worked the aci_route_control_profile module and the modules for child objects to support profiles directly created under the tenant. I've also updated the integration tests to include tests for profiles created directly under the tenant instead of under an l3out.